fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+#function fizzbuzz
+def fizzbuzz(a, b):
+    if isinstance((a and b), list):
```

Comment:
What do you expect your program to do if two empty lists are entered?

Reply:
Had not catered for this but thanks a lot @dennisja . Let me effect it